pkg/apis/migration/migration.go

```diff
@@ -108,7 +108,7 @@ var (
 	// read, write on volumeMigrationInstance.
 	volumeMigrationInstanceLock = &sync.RWMutex{}
 	// deleteVolumeInfoLock is used for handling race conditions during
-	// volumeMigration CR deletion.
+	// volumeinfo CR deletion.
```

Comment:
I guess you changed it in the wrong file

Reply:
fixed.